saleor/checkout/utils.py

```diff
@@ -828,7 +828,8 @@ def _fill_order_with_cart_data(order, cart, discounts, taxes):
             order, line.variant, line.quantity, discounts, taxes)
 
     if cart.note:
-        order.notes.create(user=order.user, content=cart.note)
+        order.customer_note = cart.note
+        order.save()
```

Comment:
if saving just one field, we could use `updated_fields` here

Reply:
We could pass this field data at creation time instead:
https://github.com/mirumee/saleor/blob/a945504c5fc1e1d7471212681761751864e8773e/saleor/checkout/utils.py#L828-L836